system-tests/docker-compose.yml

```diff
@@ -157,12 +157,14 @@ services:
       EDC_IAM_DID_WEB_USE_HTTPS: "false"
       EDC_CONNECTOR_NAME: registration-service
       EDC_IDENTITY_DID_URL: did:web:did-server:registration-service
+      EDC_SELF_DESCRIPTION_DOCUMENT_PATH: /resources/self-description/registration-service/sdd.json
       EDC_VAULT: /resources/vault/registration-service/registration-service-vault.properties
       EDC_KEYSTORE: /resources/vault/registration-service/registration-service-keystore.jks
       EDC_KEYSTORE_PASSWORD: test123
       EDC_ERROR_RESPONSE_VERBOSE: "true"
     ports:
       - "8184:8184"
+      - "8185:8181"
```

Comment:
Just for understanding...  why was this required?

Reply:
The identity hub endpoint of the registration service is used in the `IdentityHubIntegrationTest`